### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by devnook**
The package.json data is now used to populate chrome.dev meta info about demos.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "web-ai-demos",
+  "version": "1.0.0",
+  "description": "This repository contains demos related to client-side (in-browser) AI.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```
Comment (tomayac) [inline package.json:10]:
```suggestion
  "license": "Apache-2.0"
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "web-ai-demos",
+  "version": "1.0.0",
+  "description": "This repository contains demos related to client-side (in-browser) AI.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (tomayac) [inline package.json:8]:
```suggestion
```

File: product-review-suggestions/package.json [hunk 3]
```diff
@@ -8,7 +9,6 @@
   },
   "author": "",
   "license": "ISC",
```
Comment (tomayac) [inline product-review-suggestions/package.json:11]:
```suggestion
  "license": "Apache-2.0",
```

File: summarization-api-playground/package.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "summarization-api-playground",
+  "description": "A demo of Chrome's experimental Summarization API.",
```
Comment (tomayac) [inline summarization-api-playground/package.json:3]:
```suggestion
  "description": "A demo of Chrome's experimental Summarizer API.",
```

File: toxic-review-warning/package.json [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
   "name": "perf-test",
   "version": "1.0.0",
+  "description": "Demo of using AI to detect toxic reviews",
   "scripts": {
     "start": "parcel src/index.html",
     "build": "parcel build --public-url=./ src/index.html",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
   "license": "ISC",
```
Comment (tomayac) [inline toxic-review-warning/package.json:11]:
```suggestion
  "license": "Apache-2.0",
```

**Review by tomayac — APPROVED** [review]
LGTM after suggestions applied

File: package.json (hunk 2 above)
Comment (devnook) [reply package.json:8]: is that accidental?

Comment (tomayac) [reply package.json:8]: It's intended. This seems superficial as it doesn't do anything.

File: toxic-review-warning/package.json [hunk 6]
```diff
@@ -1,14 +1,15 @@
 {
   "name": "perf-test",
   "version": "1.0.0",
+  "description": "Demo of using AI to detect toxic reviews",
   "scripts": {
     "start": "parcel src/index.html",
     "build": "parcel build --public-url=./ src/index.html",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
```
Comment (maudnals) [inline toxic-review-warning/package.json:10]:
```suggestion
  "author": "Maud Nalpas",
```

File: product-review-suggestions/package.json [hunk 7]
```diff
@@ -1,14 +1,14 @@
 {
   "name": "perf-test",
+  "description": "Demo of using Gen AI to help users write informative reviews",
   "version": "1.0.0",
   "scripts": {
     "start": "parcel src/index.html",
     "build": "parcel build --public-url=./ src/index.html",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
```
Comment (maudnals) [inline product-review-suggestions/package.json:10]:
```suggestion
  "author": "Maud Nalpas",
```

File: toxic-review-warning/package.json [hunk 8]
```diff
@@ -1,14 +1,15 @@
 {
   "name": "perf-test",
```
Comment (maudnals) [inline toxic-review-warning/package.json:2]:
```suggestion
  "name": "toxic-review-warning",
```

**Pull request merged**